R/applications.R

```diff
@@ -159,7 +159,7 @@ applicationTask <- function(taskDef, appName, account, server, quiet) {
 # streams application logs from ShinyApps
 streamApplicationLogs <- function(authInfo, applicationId, entries, skip) {
   # build the URL
-  url <- paste0(shinyappsServerInfo()$url, "/applications/", applicationId,
+  url <- paste0(serverInfo("shinyapps.io")$url, "/applications/", applicationId,
```

Comment:
This should not rely on the "shinyapps.io" server name and instead rely on some target server object. We're getting away with this today because shinyapps.io and posit.cloud use the same API hostnames.

The caller (`showLogs`) has an appropriate client; maybe log streaming should be implemented there, alongside `getLogs`?

Reply:
I think this is probably missing a method on the client object, but it's going to be a while before we get around to fixing that. IMO this code is no worse that the previous, because all server logic is now routed through `serverInfo()`